### ISSUE TITLE
[pogues2lunatic] - Simplify transformation 

### ISSUE THREAD

**Issue by laurentC35** [InseeFr/Pogues-Back-Office#182]

**Comment by laurentC35** [comment]
I think we need a PoguesMapper to do this -> https://github.com/InseeFr/Eno/tree/v3-main/eno-core/src/main/java/fr/insee/eno/core/mappers

**Comment by nsenave** [comment]
@laurentC35 yes Pogues as input is the next planned in Eno java. Once done, we could do Pogues -> Lunatic directly.

For now this is paused, since we're thinking about doing a refresh of Pogues, which might change the model a lot.

**Comment by nsenave** [comment]
However, to prepare for this, I'd like to introduce a "Pogues to Lunatic" endpoint in Eno web-service, which could be useful here in Pogues backend.
